rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/jobs/JobRequest.groovy

```diff
@@ -26,5 +27,17 @@ import groovy.transform.Immutable
 @CompileStatic
 class JobRequest {
   List<String> tokenizedCommand
+  List<String> packerParameterFilters = []
+
+  List<String> getFilteredTokenizedCommand() {
+    Lists.newArrayList(tokenizedCommand).collect {
```

Comment:
What's the reason you need to use `newArrayList` if `tokenizedCommand` is already a `List`?